index.html

```diff
@@ -477,20 +511,3 @@
   </section>
 </body>
 </html>
```

Comment:
Yeah, basically, we just need to rewrite that text for each method. But I think it's worth it, because you end up with a much nicer and extensible document where each method is defined on its own. This allows you to, for example, add an example for each method, etc.